src/index.template.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title> Lif Market Validation Mechanism - WindingTree </title>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">
+    <link  rel="stylesheet" href="https://fonts.googleapis.com/css?family=Catamaran:400,500,600,700,800">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
+    <link rel="stylesheet" href="https://cdn.rawgit.com/ShakrMedia/tuesday/master/build/tuesday.min.css" />
+
+    <link rel="stylesheet" href="app.css">

```

Comment:
This shouldn't be necessary. The `HtmlWebpackPlugin` should inject CSS automatically, if there's at least one place where the main css file is required.

Reply:
Yes, removed!